core/src/main/java/org/apache/calcite/util/RelToSqlConverterUtil.java

```diff
@@ -63,6 +63,36 @@ public static void unparseHiveTrim(
     }
   }
 
+  /**
+   * Unparses Array and Map value constructor.
+   *
+   * <p>For example :
+   *
+   * <blockquote><pre>
+   * SELECT ARRAY[1, 2, 3] &rarr; SELECT ARRAY (1, 2, 3)
+   * SELECT MAP['k1', 'v1', 'k2', 'v2'] &rarr; SELECT MAP ('k1', 'v1', 'k2', 'v2')
+   * </pre></blockquote>
+   *
+   * @param writer writer
+   * @param call the call
+   */
+  public static void unparseSparkArrayAndMap(SqlWriter writer,
```

Comment:
minor: putting the unparse method in the `RelToSqlConverterUtil` tool class feels a bit ambiguous,  because unparse should belong to SqlNode to Sql rather than RelNode to SqlNode.

Reply:
Hi @YiwenWu ,I noticed this too. So how about when we merge this after , In another PR ,We can create new SqlNodeToSqlUtil , Then refactor the relative code ?

Reply:
I agree, it's a good idea